src/inflators/ambient-light.ts

```diff
@@ -0,0 +1,21 @@
+import { addComponent } from "bitecs";
+import { addObject3DComponent } from "../utils/jsx-entity";
+import { AmbientLight, LightTag } from "../bit-components";
+import { AmbientLight as AL } from "three";
+import { HubsWorld } from "../app";
+
+export type AmbientLightParams = {
+  color: string;
+  intensity: number;
+};
+
+export function inflateAmbientLight(world: HubsWorld, eid: number, params: AmbientLightParams) {
```

Comment:
In the aframe component we have default values. We probably want to preserve those. https://github.com/mozilla/hubs/blob/master/src/components/ambient-light.js#L3-L4

We can do this easily with `Object.assign()` like https://github.com/mozilla/hubs/blob/master/src/inflators/simple-water.ts#L34

Reply:
Updated, thanks